STYLE.md

```diff
@@ -25,6 +25,11 @@
   * `typedef std::shared_ptr<Bar> BarSharedPtr;`
   * `typedef std::shared_ptr<const Blah> BlahConstSharedPtr;`
   * Regular pointers (e.g. `int* foo`) should not be type aliased.
+* If move semantics are intended, prefer specifying function arguments with `&&`. 
```

Comment:
Maybe add some comment to the effect that this is even true if a `std::unique_ptr` is involved (it's in the example, but a casual reader might imagine this is a mistake).